azext_edge/__init__.py

```diff
@@ -6,20 +6,38 @@
 
 from azure.cli.core import AzCommandsLoader
 from azext_edge.constants import VERSION
+from knack.events import EVENT_INVOKER_POST_PARSE_ARGS
+
+
+def version_check_handler(cli_ctx, **kwargs):
+    from .edge.util.version_check import check_latest
+
+    command: str = kwargs.get("command")
+    if command:
+        if command.startswith("iot ops"):
+            if command == "iot ops init":
```

Comment:
I thought we were supposed to care if a person has an old deployment too (as in encourage them to upgrade + redo init)... if so then wouldn't spamming this check for every iot ops call be better? Or does the service have an answer to that already?